packages/jest-circus/src/testCaseReportHandler.ts

```diff
@@ -13,6 +13,7 @@ const testCaseReportHandler =
   (testPath: string, sendMessageToJest: TestFileEvent) =>
   (event: Circus.Event): void => {
     switch (event.name) {
+      case 'test_todo':
       case 'test_done': {
```

Comment:
Hm.. I would try testing it similar to `'test_done'`. There should be some unit test around. It does not look that you need an e2e test here. This way you could avoid bumping into Jasmine.

Reply:
It looks like jasmine gives out no output even for `test()` or anything, even a console.log() doesn't get printed to stdout. I'll look into it tomorrow, also I'm not sure if this would be in any other test file. This is the only test for custom reporters I could find.

Reply:
I was trying to say that the implementation is not reporter specific. There should be a unit test simply in `jest-circus` package.

Reply:
I'm not sure if there is any similar unit test in `jest-circus` package. Anyway I found `customReportersOnCircus.ts` right beside `customReporters.ts` which specifically skips jasmine tests made for `jest-circus`, I guess thats how it should be.